build.html

```diff
@@ -1,80 +1,80 @@
-<div class="is-loading" data-interactive-floorplan-id="{{ id }}">
-  <div v-if="containsData" class="floorplan-component-wrapper">
-    <div class="floor-wrapper-holder">
-      <div class="floor-wrapper" v-for="(floor, index) in floors" v-bind:class="{ 'active': selectedFloorData && floor.id === selectedFloorData.id && imageLoaded }">
-        <div class="floor-container" :id="'floor-' + floor.id">
-          <img v-if="floor.image" class="floor-image" :src="floor.image.url"/>
+<div class="is-loading" data-interactive-map-id="{{ id }}">
```

Comment:
How about `{{ uuid }}`?

```suggestion
<div class="is-loading" data-interactive-map-id="{{ id }}" data-interactive-map-uuid="{{ uuid }}">
```